v1/talk.yaml

```diff
@@ -0,0 +1,141 @@
+            If you would like to avoid automatically following redirect pages, set the `redirect=false` query parameter.
+
+            Beware that redirected pre-flighted cross-origin requests (such as those sending custom request headers like `Api-User-Agent`)
+            will fail in most current browsers [due to a spec bug](https://github.com/whatwg/fetch/issues/204).
+        404:
+          description: Unknown page title
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/problem'
+        default:
+          description: Error
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/problem'
+      x-request-handler:
+        - get_from_pcs:
+            request:
+              method: get
+              headers:
+                accept-language: '{{accept-language}}'
+              uri: '{{options.host}}/{domain}/v1/page/talk/{title}/{revision}'
+            return:
+              status: 200
+              headers: '{{ get_from_pcs.headers }}'
+              body: '{{get_from_pcs.body}}'
+      x-monitor: true
+      x-amples:
+        - title: Get structured talk page for enwiki Salt article
+          request:
+            params:
+              domain: en.wikipedia.org
+              title: Salt
+          response:
+            status: 200
+            headers:
+              content-type: /application\/json; charset=utf-8; profile=".+Talk.+"/
+              etag: /^"[^/"]+/[^/"]+"$/
+            body:
+              topics:
+                - id: /.+/
+                  replies:
+                    - sha: /.+/
+                      depth: /.+/
+                      html: /.*/
+                  depth: /.+/
+                  html: /.*/
+                  shas:
+                    html: /.+/
+                    indicator: /.+/
+
+  /talk/{title}/{revision}:
+    x-route-filters:
+      <<: *talk_title_revision_filter
+    get:
+      <<: *talk_title_revision_get_spec
+      parameters:
+        - name: title
+          in: path
+          description: 'Page title. Use underscores instead of spaces. Example: `Main_Page`.'
+          required: true
+          schema:
+            type: string
+        - name: revision
+          in: path
+          description: |
+            Optional page revision. Note that older revisions are not stored, so request latency with the revision would be higher.
+          required: true
+          schema:
+            type: integer
+        - name: redirect
+          in: query
+          description: |
+            Requests for [redirect pages](https://www.mediawiki.org/wiki/Help:Redirects) return HTTP 302 with a redirect target in `Location` header and content in the body.
+            To get a 200 response instead, supply `false` to the `redirect` parameter.
+          schema:
+            type: boolean
```

Comment:
this will ingerit the whole talk_title_revision_get_spec thus the x-amples too, so it will fail. we need to x-monitor "false" this one.

Reply:
Done.